lib/aida_web/controllers/session_controller.ex

```diff
@@ -41,6 +41,24 @@ defmodule AidaWeb.SessionController do
     end
   end
 
+  def forward_messages(conn, %{
+        "bot_id" => bot_id,
+        "session_id" => session_id,
+        "forward_messages_id" => forward_messages_id
+      }) do
+    session = Session.get(session_id)
+
+    if session.bot_id != bot_id do
+      conn |> put_status(422) |> json(%{errors: "Unknown session"}) |> halt
+    else
+      session
+      |> Session.put("forward_messages_id", forward_messages_id)
```

Comment:
This should be a hidden session variable (starting with `.`) to avoid clashing with user defined names (I know it's unlikely, but just to be on the safe side)